### DISCUSSION TITLE
Some Repo URLs do not open when you click on them

### DISCUSSION THREAD

**Discussion by ethndotsh** [community/community#40312] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.90.0 (84076705)
Device: iPhone 11 Pro (iOS 16.1)
</details>

Some repo URLs do not open when I click on them. One notable repository where this is happening is tw-classed (sannajammeh/tw-classed) which has a .vercel.app domain.

**Comment by mattruggio** [comment]
Hey @cursecodes!  I tried loading up that repo on an iOS device running GitHub mobile 1.92 and was able to click through the main vercel.app link along with the link in the README.  I am sure I am missing some context here, could you possibly provide a bit more info regarding the exact reproduction steps (potentially even a video demonstrating the issue)?
